samples/instances.js

```diff
@@ -15,38 +15,202 @@
 
 'use strict';
 
-function listInstances() {
-  // [START bigtable_list_instances]
-  // Imports the Google Cloud client library
-  const Bigtable = require('@google-cloud/bigtable');
+// Imports the Google Cloud client library
+const Bigtable = require('@google-cloud/bigtable');
 
-  // Creates a client
+async function runInstanceOperations(instanceName, clusterName, clusterName2) {
   const bigtable = new Bigtable();
 
-  // Lists all instances in the current project
-  bigtable
-    .getInstances()
-    .then(results => {
-      const instances = results[0];
-      console.log('Instances:');
-      instances.forEach(instance => console.log(instance.id));
-    })
-    .catch(err => {
-      console.error('ERROR:', err);
+  console.log(`\nCreating a PRODUCTION Instance`);
+  // [START bigtable_create_prod_instance]
+  // Creates a Production Instance with the name "ssd-instance"
+  // with cluster name "ssd-cluster", 3 nodes and location us-central1-f
+
+  // Set options to create an Instance
+  const instanceOptions = {
+    clusters: [
+      {
+        name: clusterName,
+        nodes: 3,
+        location: 'us-central1-f',
+        storage: 'ssd',
+      },
+    ],
+    type: 'PRODUCTION', // Optional as default type is PRODUCTION
+    labels: {'prod-label': 'prod-label'},
+  };
+
+  // Create production instance with given options
+  try {
+    const [instance] = await bigtable.createInstance(
+      instanceName,
+      instanceOptions
+    );
+
+    console.log(`\nCreated Instance: ${instance.name}`);
+  } catch (err) {
+    console.error(`Error creating prod-instance ${err}`);
+    return;
+  }
+  // [END bigtable_create_prod_instance]
+
+  console.log(`\nListing Instances\n`);
+  // [START bigtable_list_instances]
+  try {
+    const [instances] = await bigtable.getInstances();
+    instances.forEach(instance => {
+      console.log(instance.name);
     });
+  } catch (err) {
+    console.error(`Error listing instances ${err}`);
+    return;
+  }
   // [END bigtable_list_instances]
+
+  console.log(`\nGet Instance`);
+  // [START bigtable_get_instance]
+  try {
+    const [instance] = await bigtable.instance(instanceName).get();
+    console.log(`Instance Name: ${instance.name}`);
+    console.log(`Instance Meta: ${JSON.stringify(instance.metadata)}`);
+  } catch (err) {
+    console.log(`Error getting instance ${err}`);
```

Comment:
Use `console.error` and send the error object as its own argument. See https://github.com/googleapis/nodejs-bigtable/pull/139#discussion_r190997148 for more details